### PR TITLE
Add instructions on how to user a different user

### PR DESCRIPTION
**Pull request by rugk**
grafik

Unfortunately, still a little grey… but uhm… dunno, it's the official mail/creds they state in their Readme.

If you know a better way, let me know.

### CODE REVIEW

**Comment by EndBug** [comment]
That is just the picture that GitHub uses for any unknown user, so even if you use `email@example.com` the logo should be the same.  
An alternative would be using `actions@github.com`, which I think it's the official placeholder account, since it's been reserved by GItHub. As you can see from the [profile](https://github.com/actions-user), it has the actions logo but it's cropped, so I wouldn't advise using it. Also, if you use this email GitHub will always show `actions-user` as the author name, even if the commit is signed with another name.

Unless we find a better alternative, I think it's better if we just leave it as it is.

**Comment by rugk** [comment]
Well… then I wonder why they did suggest this different mail [in their own example](https://github.com/actions/checkout/#push-a-commit-using-the-built-in-token).
Whether to use `actions@github.com` or `github-actions@github.com` hardly makes a difference I guess, so if the latter shows the proper avatar, why not use `actions@github.com`?

I see nothing wrong with that. (Semantically) It's the user who committed it.

But it seems we're not the first one to have this issue: https://github.com/ad-m/github-push-action/commit/275307750b3dfa94a664de1c4a6e00dd66d90b70

The investigation there turned out your mail also does not work. It needs to be `41898282+github-actions[bot]@users.noreply.github.com`. Should I adjust that?

**Comment by EndBug** [comment]
`41898282+github-actions[bot]@users.noreply.github.com` seems great to me! If you can edit the example to feature this email it's ready to merge. Also, explain that this will show the GitHub logo ;)

File: README.md [hunk 1]
```diff
@@ -104,6 +104,21 @@ For more info on how to use outputs, see ["Context and expression syntax"](https
...
+```yaml
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: EndBug/add-and-commit@v7.0.0
+        with:
+          author_name: github-actions
+          author_email: 41898282+github-actions[bot]@users.noreply.github.co
```
Comment (EndBug) [inline README.md:117]:
```suggestion
          author_email: 41898282+github-actions[bot]@users.noreply.github.com
```

File: README.md [hunk 2]
```diff
@@ -104,6 +104,21 @@ For more info on how to use outputs, see ["Context and expression syntax"](https
...
+```
+
+The mail [`41898282+github-actions[bot]@users.noreply.github.co` is the mail] of the original GitHub Actions bot. If you use that, [the GitHub avatar is shown for the commits](https://github.community/t/github-actions-bot-email-address/17204).
```
Comment (EndBug) [inline README.md:120]:
```suggestion
`41898282+github-actions[bot]@users.noreply.github.com` is the mail of the original GitHub Actions bot. If you use that, [the GitHub avatar is shown for the commits](https://github.community/t/github-actions-bot-email-address/17204). 
```

File: README.md (hunk 1 above)
Comment (rugk) [reply README.md]: Oh sorry :see_no_evil:

**Pull request merged**